### PR TITLE
Angle bar list

### PR DESCRIPTION
**Pull request by DanGrayson**
add a new type of visible list of a type called AngleBarList
    
Example:

        i11 : <| x,y,z |>
    
        o11 = <|x, y, z|>

Intended use: associative algebras, by Stillman and Moore

### CODE REVIEW

**Comment by DanGrayson** [comment]
@mikestillman @moorewf

File: M2/configure.ac [hunk 1]
```diff
@@ -1139,8 +1139,8 @@ AC_LANG(C++)
 AC_CHECK_HEADER(frobby.h,LIBS="-lfrobby $LIBS",BUILD_frobby=yes)
 if test $BUILD_frobby = yes
 then BUILTLIBS="-lfrobby $BUILTLIBS"
-    dnl change this to [1] when frobby is built from a submodule
-    AC_DEFINE([HAVE_FROBBY_VERSION], [0], [whether frobby has frobby_version])
+    dnl uncomment this to [1] when frobby is built from a submodule (?)
+    dnl AC_DEFINE([HAVE_FROBBY_VERSION], [0], [whether frobby has frobby_version])
```
Comment (mahrud) [inline M2/configure.ac:1143]: This is not going to work with both the new and old version of frobby.

**Comment by mahrud** [comment]
Is this different from the [bra-ket notation](https://en.wikipedia.org/wiki/Bra%E2%80%93ket_notation)?

File: M2/configure.ac (hunk 1 above)
Comment (mahrud) [inline M2/configure.ac:1143]: Did you test this with both the old version and the new version of Frobby?

**Comment by DanGrayson** [comment]
> Is this different from the [bra-ket notation](https://en.wikipedia.org/wiki/Bra%E2%80%93ket_notation)?

Yes.  Bra-ket notation is the binary operator `<x|y>`, together with the convention that `<x|` and `|y>` can stand alone.

File: M2/configure.ac (hunk 1 above)
Comment (DanGrayson) [reply M2/configure.ac:1143]: There's nothing to test -- the line is active only when we are building frobby, and the version to build is hard-coded in frobby/Makefile.in.

Also, `#define HAVE_FROBBY_VERSION 0` doesn't have the desired effect on your line `#ifdef HAVE_FROBBY_VERSION` in `version.dd`.

**Pull request merged**